project.clj

```diff
@@ -8,12 +8,14 @@
   :pedantic? :abort
   :dependencies [[org.clojure/clojure "1.5.1"]
                  [org.clojure/tools.logging "0.2.6"]
+                 [clj-time "0.5.1"]
```

Comment:
We already had a transitive dependency on it, this just pins to a specific version.  For some reason IDEA started complaining that we had multiple conflicting transitive deps on it.